tools/build-toolchain.sh

```diff
@@ -38,35 +78,96 @@ command_exists () {
 
 # Download the file URL using wget or curl (depending on which is installed)
 download () {
-  if   command_exists wget ; then wget -c  "$1"
+  if   command_exists wget ; then wget --no-check-certificate -c  "$1"
```

Comment:
Why add `--no-check-certificate`?

Reply:
It turns out that wget needs extra CA dependencies (i.e. it actually checks the root CA). Try building the docker image against WGET with no CURL installed... This was an easy workaround.

Reply:
I have added a comment in source about this.